<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>746</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>746</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fan control</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="applymode_btn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>61</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Fan mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="silent_mode">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>67</y>
      <width>58</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Silent</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rb_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="normal_mode">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>67</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rb_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="gaming_mode">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>67</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaming</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rb_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="auto_mode">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>67</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rb_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="custom_mode">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>67</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rb_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="fixed_mode">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>67</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rb_buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>62</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fan modes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="applylimit_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="charging_limit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>60</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>9</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Root password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>121</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Charging limit</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rb_buttonGroup"/>
 </buttongroups>
</ui>
